understanding/identify-purpose.html

```diff
@@ -55,143 +53,115 @@
-                     <li><a href="#intent">Intent</a></li>
-                     <li><a href="#benefits">Benefits</a></li>
-                     <li><a href="#examples">Examples</a></li>
-                     <li><a href="#resources">Related Resources</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#resources">関連リソース</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 1.3.6:</span>Identify Purpose (Level AAA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 1.3.6:</span>目的の特定 (レベル AAA)
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>In content implemented using markup languages, the purpose of <a href="#dfn-user-interface-component">user interface components</a>, icons, and <a href="#dfn-region">regions</a> can be <a href="#dfn-programmatically-determined">programmatically determined</a>.
+                  <p>マークアップ言語で実装されたコンテンツでは、<a href="#dfn-user-interface-component">ユーザインタフェース コンポーネント</a>、アイコン、<a href="#dfn-region">領域</a>の目的は<a href="#dfn-programmatically-determined">プログラムによる解釈</a>が可能である。
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   				
-                  <dt>Goal</dt>
-                  <dd>It is easier to operate and navigate content.</dd>
+                  <dt>目標</dt>
+                  <dd>コンテンツを操作及びナビゲートすることを、より簡単にできるようにする。</dd>
                   				
-                  <dt>What to do</dt>
-                  <dd>Use code to indicate the meaning of all controls and other key information, where
-                     available.
+                  <dt>何をすればよいか</dt>
+                  <dd>コードを用いて、すべてのコントロールの意味、及びもしあればその他の重要な情報を示す。
```

Comment:
「, where available」の係り方がおかしいような。たとえばこう？
```suggestion
                  <dd>可能な場合、コードを用いて、全てのコントロールの意味及びその他の重要な情報を示す。
```

Reply:
あれ、ホントですね (これは明らかにおかしいです...)。ありがとうございます！